aio/content/navigation.json

```
@@ -102,6 +102,11 @@
       "title": "Understanding Angular",
       "tooltip": "Learn the concepts essential to becoming a proficient Angular developer.",
```

```suggestion
      "tooltip": "Learn the essential concepts to become a proficient Angular developer",
```

A few word changes to help simplify the tone.